### PR TITLE
when add path to toolrunner args, use tl.pathArg()

### PR DESCRIPTION
**Pull request by TingluoHuang**
Conflicts:
	Tasks/Grunt/task.json
	Tasks/Grunt/task.loc.json
	Tasks/Gulp/task.json
	Tasks/Gulp/task.loc.json
	Tasks/XamarinTestCloud/task.json
	Tasks/XamarinTestCloud/task.loc.json

### CODE REVIEW

**Comment by TingluoHuang** [comment]
Bump external vsts-task-lib version.

File: Tasks/Scvmm/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// *******************************************************
```
Comment (bryanmacfarlane) [inline Tasks/Scvmm/Strings/resources.resjson/en-US/resources.resjson]: take out of patch

File: Tasks/SqlAzureDacpacDeployment/Strings/resources.resjson/en-US/resources.resjson [hunk 2]
```diff
@@ -8,8 +8,11 @@
   "loc.instanceNameFormat": "Deploy Azure SQL DACPAC: $(DacpacFile)",
   "loc.group.displayName.target": "Target",
   "loc.group.displayName.firewall": "Firewall",
```
Comment (bryanmacfarlane) [inline Tasks/SqlAzureDacpacDeployment/Strings/resources.resjson/en-US/resources.resjson]: take out of patch

**Comment by bryanmacfarlane** [comment]
Other than notes - looks good.  Get validation from customer

**Comment by TingluoHuang** [comment]
Is the fix already in master?  Yes
What testing was done? Run gulp task, make sure with space in working folder, the task run successfully.
Does this fix change the AT/DT contract? N/A
Does this fix include any database servicing changes? N/A
Were tests added to cover this to make sure it doesn't regress?  We add L0 test to vsts-task-lib

**Comment by TingluoHuang** [comment]
Bug 507871: grunt task failed if the path to gruntfile.js has space
https://mseng.visualstudio.com/DefaultCollection/VSOnline/_workitems/edit/507871?fullScreen=false

**Pull request merged**